src/types/publication_types/artisticRegistration.types.ts

```diff
@@ -177,7 +177,10 @@ export interface MusicTrack {
 
 export interface AudioVisualPublication extends ArtisticOutputBase {
   type: 'AudioVisualPublication';
-  mediaType: MusicMediaType | '';
+  mediaType: {
+    type: MusicMediaType | '';
+    description: string | '';
```

Comment:
tom string er også string
```suggestion
    description: string;
```